packages/terra-table/src/Table.jsx

```diff
@@ -0,0 +1,352 @@
+import React, {
+  useState, useContext, useRef, useCallback, useEffect, useMemo,
+} from 'react';
+import PropTypes from 'prop-types';
+import { injectIntl } from 'react-intl';
+import classNames from 'classnames/bind';
+import ResizeObserver from 'resize-observer-polyfill';
+import ThemeContext from 'terra-theme-context';
+import VisuallyHiddenText from 'terra-visually-hidden-text';
+
+import ColumnHeader from './subcomponents/ColumnHeader';
+import Row from './subcomponents/Row';
+import rowShape from './proptypes/rowShape';
+import { columnShape } from './proptypes/columnShape';
+import ColumnContext from './utils/ColumnContext';
+import styles from './Table.module.scss';
+import GridContext from './utils/GridContext';
+import validateRowHeaderIndex from './proptypes/validators';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /**
+   * String that identifies the element (or elements) that labels the table.
+   */
+  ariaLabelledBy: PropTypes.string,
+
+  /**
+   * String that labels the table for accessibility. If ariaLabelledBy is specified, ariaLabel will not be used.
+   */
+  ariaLabel: PropTypes.string,
+
+  /**
+   * String that will be used to identify the table. If multiple tables are on the same page, each table should have
+   * a unique id.
+   */
+  id: PropTypes.string.isRequired,
+
+  /**
+   * Prop that determines whether the table's container should be scrollable.
+   */
+  isScrollable: PropTypes.bool,
```

Comment:
I don't think consumers should have to pass a prop for if the table is scrollable.

Reply:
This was something I tried to do and somehow I forgot to remove this. It is now gone.